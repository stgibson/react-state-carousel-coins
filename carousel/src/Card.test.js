import React from "react";
import { render } from "@testing-library/react";
import Card from "./Card";
import image1 from "./image1.jpg";

it("loads without crashing", function() {
  render(
    <Card
      caption="test-caption"
      src={ image1 }
      currNum={ 1 }
      totalNum={ 2 }
    />
  );
});

it("matches snapshot", function() {
  const { asFragment } = render(
    <Card
      caption="test-caption"
      src={ image1 }
      currNum={ 1 }
      totalNum={ 2 }
    />
  );
  expect(asFragment()).toMatchSnapshot();
});