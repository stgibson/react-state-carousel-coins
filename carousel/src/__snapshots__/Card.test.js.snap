// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      test-caption
    </h4>
    <img
      alt="test-caption"
      class="Card-image"
      src="image1.jpg"
    />
    <small
      class="Card-small"
    >
      Image 1 of 2.
    </small>
  </div>
</DocumentFragment>
`;
